conjure-java-core/src/test/resources/example-types.yml

```
@@ -261,7 +261,8 @@ types:
           rids: map<RidAliasExample, ManyFieldExample>
           bearertokens: map<BearerTokenAliasExample, ManyFieldExample>
           integers: map<IntegerAliasExample, ManyFieldExample>
-          # doubles: map<DoubleAliasExample, ManyFieldExample> # typescript freaks out with the 'NaN'
+          doubles: map<DoubleAliasExample, ManyFieldExample> # typescript freaks out with the 'NaN'
```

nit: I would probably just remove the typescript comment too, as I'm unclear what value it provides in the conjure-java repo
```suggestion
          doubles: map<DoubleAliasExample, ManyFieldExample>
```